### PR TITLE
Allow alternate tau dir for tests

### PR DESCRIPTION
**Pull request by wspear**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ may fix the problem.
 
 ``` shell
 $ docker pull paratools/salt-dev:latest
-$ docker run -it --pull --tmpfs=/dev/shm:rw,nosuid,nodev,exec \
+$ docker run -it --pull always --tmpfs=/dev/shm:rw,nosuid,nodev,exec \
```
Comment (zbeekman) [inline README.md:54]: @wspear what was the motivation for changing this?

File: README.md [hunk 2]
```diff
@@ -106,11 +106,20 @@ The tests are all located in the `tests` subdirectory of the project.
...
  * New profiles are moved to subdirectories indicating which test they are assosciated with
 
-A limitation of the tests is that they assume a TAU installation matching the [salt-dev]
+By default the tests assume a TAU installation matching the [salt-dev]
 development image.
 If you don't have a GCC and Clang configuration of TAU installed into the default location
 (`/usr/local/x86_64/`) then some of the tests will fail.
 
+Alternatively you can add `-DTAU_ROOT=<dir>` to the cmake invocation to point the tests
+to a different location. The tests require TAU to be built with these two configurations:
```
Comment (zbeekman) [inline README.md:115]: So the tests pass when setting -DTAU_ROOT? Perhaps we should update the paragraph above too. Right not is reads like:

>This doesn't work. Ok, or you can try this...

I.e. a bit self contradictory. I can help with wording.

**Review by zbeekman — COMMENTED** [review]
@wspear this generally looks good. After responding to my quick questions I'll approve and merge or we can make some small tweaks if needed.

File: README.md (hunk 1 above)
Comment (wspear) [reply README.md:54]: At least as of docker 25.0.3 --pull requires an argument or else you get: `invalid pull option: '--tmpfs=/dev/shm:rw,nosuid,nodev,exec': must be one of "always", "missing" or "never".`

**Review by zbeekman — APPROVED** [review]
LGTM assuming tests all pass

**Pull request merged**